src/main.ts

```diff
@@ -1,10 +1,11 @@
-import { NestFactory } from '@nestjs/core';
-import { AppModule } from './app.module';
+import { NestFactory } from "@nestjs/core";
+import { AppModule } from "./app.module";
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
-  app.enableCors();
-  await app.listen(process.env.PORT);
+  app.setGlobalPrefix("api");
+
+  await app.listen(3000);
```

Comment:
I see that the `PORT` is still used as an environment variable. We should either keep using that here, or remove it from the component config